src/main/webapp/admin_dashboard.html

```diff
@@ -88,5 +88,19 @@ <h2> Last 10 User Tasks</h2>
         </div>
       </main>
     </div>
+    <div id="popup-overlay">
+		<div class="popup">
+			<h2>Edit Info for Task</h2>
+			<form action="">
+				<input type="text" placeholder="Task Details" id="task-details">
+
+				<input type="text" placeholder="Category" id="category">
```

Comment:
For a future PR: it's still good to give these elements `name` attributes so that the form works semantically. Even if you're just getting their values in the JavaScript by referencing the IDs of the elements.